cypress/support/pages/roles.ts

```diff
@@ -0,0 +1,47 @@
+import { keys } from "cypress/types/lodash";
+import { verify } from "cypress/types/sinon";
+import { randomNumber, role, smallRandomNumber, team, user } from "../../constants/globalConstants"
+
+//general data
+
+const name_1 = 'EA';
+const role_name = name_1+randomNumber;
+
+
+
+ //create a role
+Cypress.Commands.add('createARole', () => {
+  
+   //we are creating a super admin role
+
+   cy.get('#roles-link').click();
+   cy.get('#root > section > main > div > div > div > div > div.ant-table-title > header > div.cta-btns > a').click();
+   const name = 'EA' +randomNumber
+   cy.get('#name').type(role_name);
+   cy.get('#displayName').type(role_name);
+   cy.get('.ant-select-selection-item').type('{downarrow}');
+   cy.xpath('/html/body/div[3]/div/div/div/div[2]/div[1]/div/div/div[1]/div').click();
```

Comment:
xpath is extremely slow and expensive, and usually a very very last resort thing to use. Like in the above comment, the necessity for this is usually a sign of inadequate identifiers! developers should definitely work with you to help prevent selectors like this